recipes/jupyter-fs/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "jupyter-fs" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5e80ac5ffaa71952cdbb51a7494580183664c3a509842ff35ae83cf798ec4fda 
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - nodejs
```

Comment:
```suggestion
```

Reply:
node is both a build requirement for making the bdist and wheel, and is also needed the first time you run JupyterLab after installing this extension. What section should it go under?